articles/150_ros_command_line.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: ROS Command Line
+permalink: articles/ros_command_line.html
+abstract:
+  This article describes ROS 2 nodes command line capabilities and its syntax.
+author: '[Michel Hidalgo](https://github.com/hidmic)'
+published: true
+---
+
+- This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+## Overview
+
+As it was the case in ROS 1, ROS 2 nodes allow configuration via command line arguments to a certain degree.
+However, a larger set of configuration options, an ambiguity in remapping rules and parameter assignment syntax (as a result of the leading underscore name convention for hidden resources), a one-to-many relationship between executables and nodes, among others, add up complexity to an otherwise simple interface.
```

Comment:
I'd reword this to have the conclusion then followed by the reasons (just because it's hard to see how the enumerated items are related which reading).

Something like this (might need more polish, but just an example of flipping the order of the sentence):

> However, in ROS 2 the complexity of these command line arguments have increased due to: a larger set of configuration options, an ambiguity in remapping rules and parameter assignment syntax (as a result of the leading underscore name convention for hidden resources), a one-to-many relationship between executables and nodes, etc.

Reply:
Reworded in beaf7bec3dfc45db907f8c89e30801dcfdc3eba1.